app/controllers/Form6010/LettingOtherPartOfPropertyDetailsController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.Form6010
+
+import config.AppConfig
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.Form6010.lettingOtherPartOfPropertyDetails
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+
+@Singleton
+class LettingOtherPartOfPropertyDetailsController @Inject()(
+  mcc: MessagesControllerComponents,
+  appConfig: AppConfig,
+  lettingOtherPartOfPropertyDetails: lettingOtherPartOfPropertyDetails)
+  extends FrontendController(mcc) {
+
+  def show: Action[AnyContent] = Action.async { implicit request =>
```

Comment:
If you don't have method in Action that returns Future you can use Action instead of Action.async. then you don't need extra wrapper Future.successful().